src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -316,6 +316,12 @@
               <PaperFiling class="mt-6" />
             </template>
 
+            <!-- is this a consent to continuation out filing? -->
+            <template v-else-if="isTypeConsentContinuationOut(filing)">
+              <p class="mt-4">This consent is valid <strong>until {{filing.expiry}}</strong>.</p>
+              <p class="mt-4">{{filing.comment}}</p>
+            </template>
```

Comment:
It looks like this (except there will be a receipt document and the "Download All" button will automatically be displayed). Ethan has approved this.

![image](https://user-images.githubusercontent.com/10002889/216710155-810f2af0-f41a-4e45-9f08-c7192e05fa0f.png)